Explorer/Assets/DCL/UI/GenericContextMenu/Controls/Configs/ContextMenuControlSettings.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace DCL.UI.GenericContextMenu.Controls.Configs
+{
+    public class ContextMenuControlSettings
+    {
+        internal Delegate callback;
```

Comment:
It should be done via a binding, otherwise there is another point of failure because of the unknown set of arguments required. And it should be bound to the actual `initialValue` <-> `viewType` <-> `callback`